app/bundles/EmailBundle/Helper/MailHelper.php

```diff
@@ -238,6 +236,7 @@ public function __construct(
         private Environment $twig,
         private ThemeHelper $themeHelper,
         private SlotsHelper $slotsHelper,
+        protected EventDispatcherInterface $dispatcher,
```

Comment:
Do you think there is any reason for `protected` property here? I know it used to be like this but I think we can improve it when doing this change.
```suggestion
        private EventDispatcherInterface $dispatcher,
```